### DISCUSSION TITLE
could we have an env var for a custom config file path for each linter?

### DISCUSSION THREAD

**Discussion by NicolasMassart** [super-linter/super-linter#2608] [Ideas]
Some have a file name env, some have none.
I would love the OpenAPI linter (Spectral) to be able to have it's config file name configured because the default one is not even the same default as in the base Spectral (and there's no doc on this, I had to check the code)
Having a variable would then help using our preferred name, including the one that makes sense by the original linter default.

**Comment by ghost** [comment]
Great request. I am also confused by the conflicting advice between super-linter and spectral on which file to use to make a custom rule set, .openapirc.yml or .spectral.yml. I was going to post a question along these lines, but maybe you could help? Which is it? I have put my changes in both but the rule which I am trying to ignore stills comes up. I am running this in an Azure DevOps CI pipeline.

**Comment by ferrarimarco** [comment]
Please, open a new feature request so we can get more details about this. Thanks!
